src/components/vm/vmDetailsPage.jsx

```diff
@@ -86,19 +86,39 @@ export const VmDetailsPage = ({
     );
 
     if (cockpit.location.path[1] == "console") {
-        return (
-            <WithDialogs key="vm-details">
+        const card = (
+            <ConsoleCard
+                state={consoleState}
+                vm={vm}
+                config={config}
+                onAddErrorNotification={onAddErrorNotification}
+                isExpanded />
+        );
+
+        let page;
+        if (!window.parent || window.parent.features?.fullscreen_via_manifest) {
+            // No Shell UI around us
+            page = (
+                <div id={"vm-" + vm.name + "-consoles-page"}
+                     className="consoles-page-expanded shell-fullscreen">
+                    {card}
+                </div>
```

Comment:
These 5 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2229-7619af68-20250731-141118-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)